### PR TITLE
Add example for LoggingHandler

### PR DESCRIPTION
**Pull request by AndyHaskell**

### CODE REVIEW

File: handlers.go [hunk 1]
```diff
@@ -309,6 +309,16 @@ func CombinedLoggingHandler(out io.Writer, h http.Handler) http.Handler {
...
 //
 // LoggingHandler always sets the ident field of the log to -
+//
+// Example:
+//
+//  r := mux.NewRouter()
+//  r.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+//  	w.Write([]byte("This is a catch-all route"))
+//  })
+//  logAndServe := handlers.LoggingHandler(os.Stdout, r)
```
Comment (elithrar) [inline handlers.go]: I would name this `loggedRouter` as it's not actually serving anything (i.e. there's no `Serve()` method involved).

**Comment by elithrar** [comment]
Thanks for this @AndyHaskell - just one comment on the naming but otherwise :+1:

File: handlers.go (hunk 1 above)
Comment (AndyHaskell) [inline handlers.go]: Thanks for the naming idea. This example actually came from one of my tutorials on github.com/AndyHaskell/MEAN-Gopher where it's named logAndServe there, so if I give you a shoutout for the name there, mind if I update the name of that handler to loggedRouter in my own tutorial for more accuracy?

**Comment by AndyHaskell** [comment]
Would you mind if I change the name from logAndServe to loggedRouter in my own tutorial I originally wrote that example for if I give you a shoutout for pointing out that loggedRouter is a more accurate name?

**Pull request merged**